Languages/English/Keyed/Keys.xml

```diff
@@ -1,29 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <LanguageData>
 
-	<koisama.Numbers.Title>Numbers overview</koisama.Numbers.Title>
-	<koisama.Numbers.Count>Row Count</koisama.Numbers.Count>
-	<koisama.Numbers.ClickToToggle>Click to select pawn type.</koisama.Numbers.ClickToToggle>
+	<koisama.Numbers.Title>Numbers!</koisama.Numbers.Title>
+	<koisama.Numbers.Count>Number of rows</koisama.Numbers.Count>
```

Comment:
Number of pawns, maybe? I'm hesitant to use the word Pawns since it's a modder's word but it's also a common name.

Reply:
Hence my reluctance to use the word Pawns. It's not used in the game itself, but it's used a lot in the code.

A pawn in RimWorld is like a unit in StarCraft, or a mob in minecraft. It's hard to give an exact definition for, except maybe "any living thing that has the potential to move." In practical terms, it's people (colonists, enemies, friendlies), mechanoids and animals (including those evil megaspiders)